src/multiformats/index.ts

```diff
@@ -0,0 +1,4 @@
+import * as codec from './codecs'
+import { hasher } from './hashes'
```

Comment:
it misses a decode function that may be good to have. It could give back the swarm hash of the referenced BeeSon